libs/langgraph/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -244,6 +240,7 @@ def create_react_agent(
     interrupt_before: Optional[list[str]] = None,
     interrupt_after: Optional[list[str]] = None,
     debug: bool = False,
+    **deprecated_kwargs: dict[str, Any],
```

Comment:
```suggestion
    **deprecated_kwargs: Any,
```
believe the typing for a `**...` field automatically represents the type of the values (the keys have to be strings and the whole thing has to be a dict)

Reply:
nice will update